pubspec.yaml

```diff
@@ -9,4 +9,4 @@ description: >
   This library declares readable symbolic names for character codes.
 homepage: https://github.com/dart-lang/charcode
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.0.0 <2.0.0-dev.infinity'
```

Comment:
Don't rock the boat. I hear the comment from @lrhn , but I'd rather not have to special case packages.